### DISCUSSION TITLE
showing request path like expressjs

### DISCUSSION THREAD

**Discussion by sqllyw** [honojs/hono#2223] [General]
hi,
as a hono newbie, I'd like to know if there is a default support in hono that shows every request path like expressjs? or there is already a middleware for that? thanks

**Comment by EdamAme-x** [comment]
There used to be a `app.showRoutes` method, but it has been removed. (Deprecated)
you should get array of path from app.routes
or you can try
`https://hono.dev/helpers/dev`

**Reply by EdamAme-x** [reply]
I recommend the latter.

**Comment by sqllyw** [comment]
Thanks for al the replies, logger is the one i'm looking for:
https://hono.dev/middleware/builtin/logger
